favicon.go

```diff
@@ -20,7 +20,7 @@ func faviconHandler(cacheTime time.Duration) (func(w http.ResponseWriter, r *htt
 	}
 
 	return func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Set("Content-Type", "image/x-icon")
+		w.Header().Set("Content-Type", "image/svg+xml")
```

Comment:
Amazing times, I didn't know SVG was supported as favicon :rocket:

Reply:
I was testing this usage to try avoid converting the file to another format.
I've just checked that Firefox and Safari do not support it 😞 
An alternative it would be convert that svg file to png.